react-stopwatch-component/src/stopwatch.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+
+export default function Stopwatch() {
+  const [play, setPlay] = useState(false);
+  const [time, setTime] = useState(0);
+  const [intervalID, setIntervalID] = useState(0);
```

Comment:
Since `intervalID` is returned by `setInterval`, it's best to use `undefined` to indicate it hasn't been set. Since `undefined` is the default, you can just call `useState()`.